roadmap/implementers-guide/src/runtime/disputes.md

```diff
@@ -123,4 +123,19 @@ applied:
    as a measure not to import one sided disputes. A dispute is raised only if there are two opposing votes so if the
    client is not sending them the dispute is a potential spam.
 4. Remove unconfirmed disputes - if a dispute contains less votes than the byzantine threshold it is removed. This is
-   also a spam precaution. A legitimate client will send only confirmed disputes to the runtime.
\ No newline at end of file
+   also a spam precaution. A legitimate client will send only confirmed disputes to the runtime.
+
+# Rewards and slashing
+
+After the disputes are filtered the validators participating in the disputes are rewarded and more importantly the
+offenders are slashed. Generally there can be two types of punishments:
+* "against valid" - the offender claimed that a valid candidate is invalid.
+* "for invalid" - the offender claimed that an invalid candidate is valid.
```

Comment:
Not sure it's worth mentioning such details at the moment. We might end up not using slashing for "for invalid" disputes and use "negative era points" instead. We'll probably have a full page for slashing eventually once the design is finalized. Also this section is named "Rewards and slashing", but no mention of rewards here.

Reply:
I am in favor of keeping details for now as it shows current design and obsolete this once we have the design ironed out and that new page reflecting it.

Reply:
Agreed, we can adjust anytime.